src/server/boot/03-participant.js

```diff
@@ -21,7 +21,17 @@ export default function (app) {
           participant.participantId,
           'find',
         )
-        res.json(participant)
+
+        // ensure that presenceHistory.author does
```

Comment:
you could mention that this comment refers to the use of the toJSON methods below

Reply:
Improved the comment a bit, thanks